PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,39 @@
+<!--- Provide a general summary of your changes in the Title above -->
+
+## Types of changes
+<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+
+Note that if your changes mix up many contexts
```

Comment:
Maybe we can have "Note that a Pull Request should only contain one of refactoring, new features or documentation changes. Please separate these changes and send us 
other PRs for each. For more information on how to create a good pull request, please refer to https://medium.com/@hugooodias/the-anatomy-of-a-perfect-pull-request-567382bb6067